src/frontend/mosh-server.cc

```diff
@@ -789,34 +798,22 @@ static void serve( int host_fd, Terminal::Complete &terminal, ServerConnection &
 	      throw NetworkException( std::string( "serve: getnameinfo: " ) + gai_strerror( errcode ), 0 );
 	    }
 
+        #ifdef HAVE_UTEMPTER
```

Comment:
The refactoring has made it obvious that the syslog code doesn't log lost connectivity.  I'm guessing you don't need this for your auditing needs, but would you consider logging lost connectivity as well?

Reply:
...and we might want to log connection close as well, at the end of `run_server()`.